hack/templates/resources/90-shoot.yaml.tpl

```diff
@@ -223,6 +223,11 @@ spec:
   #     groupsClaim: groups-claim
   #     groupsPrefix: groups-prefix
   #     issuerURL: https://identity.example.com
+  #     usernameClaim: username-claim
+  #     usernamePrefix: username-prefix
```

Comment:
`usernamePrefix` seems to be duplicated now (see line 235). Is there any reason?